cdap-watchdog/src/test/java/co/cask/cdap/logging/clean/FileMetadataCleanerTest.java

```diff
@@ -305,13 +300,13 @@ public void testFileMetadataWithCommonContextPrefix() throws Exception {
   private boolean deleteAllMetaEntries(TransactionRunner transactionRunner) {
     final List<DeletedEntry> deletedEntries = new ArrayList<>();
     try {
+      int batch = 100;
       TransactionRunners.run(transactionRunner, context -> {
-        Stopwatch stopwatch = new Stopwatch();
-        stopwatch.start();
+        int count = 0;
         StructuredTable table = context.getTable(StoreDefinition.LogFileMetaStore.LOG_FILE_META);
         try (CloseableIterator<StructuredRow> iter = table.scan(Range.all(), Integer.MAX_VALUE)) {
           while (iter.hasNext()) {
-            if (stopwatch.elapsedTime(TimeUnit.SECONDS) > CUTOFF_TIME_TRANSACTION) {
+            if (count == batch) {
```

Comment:
In the test we can delete all entries in one shot, right? I don't think the test will create millions of entries